src/components/common/Notifications.js

```diff
@@ -4,4 +4,4 @@ class Notifications {
     static notify = (msg, type) => toast(msg, {autoClose: 5000, type: type, position: "top-center"});
 }
 
-export default Notifications;
+export default Notifications;
```

Comment:
```suggestion
export default Notifications;

```
Keep trailing newline